content/en/events/upcoming-events/gsoc-proposal-template.md

```diff
@@ -0,0 +1,66 @@
+# Project Title
+
+[Provide a exact same title for your project from the proposals page; including the project number as is]
+
+## Summary
+
+[Offer a brief summary of your project, highlighting the problem it addresses, your proposed solution, and the anticipated impact.]
+
+## Personal Information
+
+- **Full Name**: [Your Full Name]
+- **Email Address**: [Your Email Address]
+- **GitHub Profile**: [Your GitHub Profile URL]
+- **LinkedIn**: [Your LinkedIn Profile URL]
+- **University/College**: [Your University/College Name]
+- **Degree Program**: [Your Degree Program]
+- **Year of Study**: [Your Current Year of Study]
+- **Country of Residence**: [Your Country]
```

Comment:
Timezone? could help align with each different mentor. not sure how is usually managed

Reply:
great suggestion!